source/docs/software/wpilib-tools/robot-simulation/simulation-gui.rst

```diff
@@ -0,0 +1,120 @@
+Simulation User Interface
+=========================
+
+WPILib has extended HAL simulation to introduce a graphical user interface (GUI) component. This allows teams to easily visualize their robots inputs and outputs.
+
+Enabling the GUI
+----------------
+
+Currently, the only way to enable this feature is to modify your ``build.gradle`` file located at the root of your robot project. Simply add the below line into the ``dependencies {}`` section of your build.gradle.
+
+.. code-block:: groovy
+
+   simulation wpi.deps.sim.gui(wpi.platforms.desktop, false)
+
+.. important:: C++ requires that you change the boolean ``false`` to ``true``.
```

Comment:
Would this be better done using code-tabs?

Reply:
No, because then you'll have two duplicate "groovy" tabs.